app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :user_games
+  has_many :games, through: :user_games
+  has_many :friendships
+  has_many :friends, through: :friendships, source: :friend
+
+  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
```

Comment:
dang dude - solid db chops. Ensures we can call friends through from the friend_id